torch/_torch_docs.py

```diff
@@ -1752,7 +1752,8 @@ def merge_dicts(*dicts):
            r"""
 conj(input, *, out=None) -> Tensor
 
-Computes the element-wise conjugate of the given :attr:`input` tensor.
+Computes the element-wise conjugate of the given :attr:`input` tensor. If the input has a non-complex dtype,
```

Comment:
Would: "Computes the element-wise complex conjugate of :attr\`input\`." be a clearer fist sentence? "Conjugate" seems overloaded.

I wonder if the second sentence shouldn't be a note and/or a warning? Maybe something like:

NOTE: if :attr\`input\` is a non-complex dtype this function just returns  :attr:`input`.

WARNING: In the future torch.conj may return a non-writeable view of :attr:`input`. It's recommended that programs not modify the tensor returned by torch.conj to be compatible with this change.

Reply:
I think the behavior for non-complex tensors should go in the description, but added a warning as per your suggestion.